soh/soh/Enhancements/pausewarp.c

```diff
@@ -87,6 +87,47 @@ void PauseWarp_HandleSelection() {
         int song = gPlayState->pauseCtx.cursorPoint[PAUSE_QUEST];
         if (aButtonPressed && CHECK_QUEST_ITEM(song) && song >= QUEST_SONG_MINUET && song <= QUEST_SONG_PRELUDE &&
             gPlayState->pauseCtx.pageIndex == PAUSE_QUEST && gPlayState->pauseCtx.state == 6) {
+            if (gSaveContext.ship.quest.id == QUEST_RANDOMIZER) {
+                bool canplay = false;
+                switch (song) {
```

Comment:
On top of checking for randomizer, this should also have a check for ocarina button shuffle being on.

Reply:
don't think it's necessary given VB stuff seemed to skip checking setting, but added since makes sense